platforms/core-configuration/configuration-problems-base/src/main/kotlin/org/gradle/internal/cc/impl/problems/HtmlReportTemplate.kt

```diff
@@ -20,13 +20,13 @@ import java.io.BufferedReader
 import java.net.URL
 
 
-internal
-object HtmlReportTemplate {
+class HtmlReportTemplate(private val reportHtmlFileName: String = "configuration-cache-report.html") {
 
-    const val reportHtmlFileName = "configuration-cache-report.html"
+    val htmlFileName: String
```

Comment:
❓ Why not expose the property directly from the constructor?

Reply:
Removed it, wasn't used :)